src/Runner/FileCachingLintingIterator.php

```diff
@@ -24,13 +24,13 @@
  */
 final class FileCachingLintingIterator extends \CachingIterator
 {
+    private LinterInterface $linter;
+
     /**
      * @var LintingResultInterface
      */
     private $currentResult;
```

Comment:
```suggestion
    private LintingResultInterface $currentResult;
```

Reply:
Maybe lets do these in another PR later, I'm not 100% what is going on in this iterator and so prefer not to change is.